base/int.jl

```diff
@@ -174,8 +174,15 @@ julia> abs(-3)
 julia> abs(1 + im)
 1.4142135623730951
 
-julia> abs(typemin(Int64))
--9223372036854775808
+julia> abs.(typemin(Int8):Int8(-125))  # overflow at typemin
+4-element Vector{Int8}:
+ -128
+  127
+  126
+  125
+
+julia> maximum(abs, [1, -2, 3, -4])
```

Comment:
I don't really see what extra information this example provides? From the earlier example we already know that `-4` turns into `4` and an example saying what is the max number of `[1,2,3,4]` seems redundant?

Reply:
It's just an attempt to have one related neat thing you may not know about.